README.md

```diff
@@ -1318,13 +1318,56 @@ ingress:
 * Description: If you use an image that is not public. then you can create dockerconfigjson secrets on your cluster and
   reference them here.
 
+### Elastic
+
+#### `elastic.enabled`
+
+* Required: false
+* Default: `false`
+* Description: Enable or disable elastic search subsystem
+> [!IMPORTANT]  
+> Alfresco can only use one search subsystem at a time, if you still have SOLR enabled (which you -shouldn't- when enabling this option),
+> SOLR will take precedence in the alfresco configmap.
+
+#### `elastic.host`
+
+* Required: false
+* Default: `elasticsearch`
+* Description: Host (service) name of elastic.
+
+#### `elastic.port`
+
+* Required: false
+* Default: `9200`
+* Description: Port of the elastic service.
+
+#### `elastic.createIndexIfNotExists`
+
+* Required: false
+* Default: `true`
+* Description: When enabled, and if the elastic index does not exist, it will be created automatically.
+
+#### `global.elasticsearch.indexName`
+
+* Required: false
+* Default: `"alfresco-{{.Release.Name}}"`
+* Description: This is a global setting for the indexname. The default name is dynamic, it is based on your release name. 
+* This prevents potential issues when using a shared elastic environment.
```

Comment:
`.Release.Name` might not be enough to ensure uniqueness.

A release name is unique within a namespace, but not globally in a kubernetes cluster.

Reply:
Good to know,
I can also take the namespace, might be better.
The only reason for this complex logic is to prevent someone (by default) writing to the same elastic index on a shared instance (which seems to be the direction we are going in...)
I will note that there are safeguards built in that prevents reindexing different alfresco instances on the same index. 

I'll alter it for namespace